src/main/java/bio/terra/app/controller/TooManyRequestsException.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.app.controller;
+
+/**
+ * This exception maps to HttpStatus.TOO_MANY_REQUESTS in the GlobalExceptionHandler.
+ */
+public class TooManyRequestsException extends Throwable {
```

Comment:
IIRC, our patterns has been to make this an abstract class and then extend it with the specific error being thrown